pyproject.toml

```diff
@@ -1,23 +0,0 @@
-[tool.black]
```

Comment:
Why delete this file?

Reply:
I don't know either, maybe miss click lol.